MapleServer2/Packets/LoginResultPacket.cs

```diff
@@ -6,10 +6,46 @@ namespace MapleServer2.Packets
 {
     public static class LoginResultPacket
     {
+        private enum LoginModes : byte
+        {
+            LoginOk = 0x00, // Success, continues login
+            IncorrectId = 0x01, // Incorrect id
+            IncorrectId2 = 0x02, // Incorrect id
+            IncorrectPassword = 0x03, // Incorrect password
+            AccountAlreadyLoggedIn = 0x04, // Account already logged in. Previous session will be aborted, clicking ok trys to relog
+            AccountAlreadyLoggedIn2 = 0x05, // Account already logged in. Previous session will be aborted, clicking ok trys to relog
+            MemberLimit = 0x06, // Member limit reached
+            AccountSuspended = 0x07, // Account suspended, shows date and reason.
+            SystemErrorDB = 0x08, // System Error: DB
+            SystemError = 0x09, // System Error
+            ChannelFull = 0x0A, // Channel is full
+            Blank = 0x0B, // Blank message
+            Blank2 = 0x0C, // Blank Message, clicking OK closes client.
+            SystemError2 = 0x0D, // System error
+            AccountInProtectionMode = 0x0E, // Account is in protection mode because it is suspected of being unauthorized. Clicking ok closes client and opens nxn website
+            IpBanned = 0x0F, // Connection is restriced for this IP. A connection is not possible. Clicking ok closes client.
+            GameVerificationFailed = 0x10, // Game verification has failed. Please try again later.
+            SystemError3 = 0x11, // System error
+            AccountNotInBeta = 0x12, // This account cannot participate in the test. A connection is not possible. Clicking ok closes client.
+            AccountNotVerified = 0x13, // Currently, this service is only available for players whose identities have been vefiried. Clicking ok closes client.
+            AccountOld = 0x14, // This account is an old account. An old account must complete identity verification at least once. Clicking ok closes client and opens nxn website
+            SystemVerificationFailed = 0x15, // Session verification failed. A connection is not possible. Clicking ok closes client.
+            GameVerificationFailed2 = 0x16, // Game verification failed. Please restart the game.
+            SystemError4 = 0x17, // System error
+            IpCannotConned = 0x18, // Connecting from an IP that cannot connect. Clicking ok closes client.
+            LoginSessionTimeout = 0x19, // Your login session has timed out. Please restart your client.
+            BuyFoundersPack = 0x1A, // Access this by purchasing a Founder's Pack.
+            SystemError5 = 0x1B, // System error
```

Comment:
This will probably be the default when neither all the other options are provided. Not sure if add it or no.